lib/generator.js

```diff
@@ -97,10 +97,12 @@ module.exports = function fixtureGenerator( options, cb ) {
    */
   function extractFixturesData( data ) {
     return Promise.map( data.section(), function( section ) {
+      var dir = path.parse( data.data.files[0] ).dir,
+          markupPath = dir.indexOf( '/partials' ) > -1 ? (dir.split('/partials')[0] + '/' + section.markup()) : (dir + '/' + section.markup());
```

Comment:
@jasonbellamy it's checking if `/partials` is part of the markup path, and if it is it removes `/partials` before adding the rest of the markup path from the KSS comments. How can I make this better because this will probably break if someone names their module `partials`?